stacker/actions/build.py

```diff
@@ -166,10 +166,19 @@ def build_parameters(self, stack, provider_stack=None):
         parameters = self._handle_missing_parameters(parameters,
                                                      required_params,
                                                      provider_stack)
-        return [
+
+        # Handle values that shouldn't be sent ('UsePreviousValue')
+        final_list = []
+        for p in [
```

Comment:
can we break this up to something like:

```
final_list = []
aws_params = [{"ParameterKey" ... for p in parameters]
for parameter in aws_params:
    ...
```

i'm not a fan of the list comp within the for statement